boat-engine/src/main/java/com/backbase/oss/boat/transformers/AdditionalPropertiesAdder.java

```diff
@@ -16,7 +16,7 @@
     private final String additionsType;
 
     @Override
-    public void transform(OpenAPI openAPI, Map<String, Object> options) {
+    public OpenAPI transform(OpenAPI openAPI, Map<String, Object> options) {
```

Comment:
Transformation happen in value by reference right? Why are you returningt the object?